ovos_gui_plugin_shell_companion/res/ui/settings/display_settings.qml

```diff
@@ -161,6 +161,13 @@ Item {
                     onClicked: {
                         console.log(autoWallpaperRotationSwitch.checked)
                         Mycroft.SoundEffects.playClickedSound(Qt.resolvedUrl("../snd/clicked.wav"))
+                    if (autoWallpaperRotationSwitch.checked) {
+                        Mycroft.MycroftController.sendRequest("ovos.wallpaper.manager.enable.auto.rotation")
+                    }
+                    else {
+                        Mycroft.MycroftController.sendRequest("ovos.wallpaper.manager.disable.auto.rotation")
+                    }
+                        // TODO: deprecate this event?
```

Comment:
i think there is no point keeping it, just messagebus spam. this PR is refactoring out the only place it was used

Reply:
It does write to `local_display_config` in the handler, but that can likely be refactored out too. Makes no sense to consume that here since it's all delegated to the PHAL plugin